### PR TITLE
fix pmcmc date bug

### PR DESCRIPTION
**Pull request by lwhittles**
remove date offsetting and deal with dates as 'days since 2019-12-31' in Markov chain

### CODE REVIEW

**Comment by johnlees** [comment]
Do you think we should just remove the offset transformation in the MCMC entirely?

File: R/pmcmc.R [hunk 1]
```diff
@@ -115,12 +115,11 @@ pmcmc <- function(data,
                                           0,
                                           0,
                                           0,
-                                          0,
                                           0),
                                     max=c(1,
                                           1,
                                           1,
-                                          1e6,
+                                          lubridate::yday("2020-03-15"),
```
Comment (johnlees) [inline R/pmcmc.R:122]: Use `sircovid_date()` instead

File: R/pmcmc.R [hunk 2]
```diff
@@ -640,9 +637,8 @@ summary.pmcmc <- function(object, ...) {
   
   sds <- round(apply(traces, 2, sd), 3)
   # convert start_date back into dates
-  summ$start_date <- offset_to_start_date(data_start_date, summ$start_date)
-  summ$start_date <- as.Date(summ$start_date, origin="2019-12-31")
-  summ[c('2.5%', '97.5%', 'min', 'max'), 'start_date'] <- summ[c('97.5%', '2.5%', 'max', 'min'), 'start_date']
```
Comment (johnlees) [inline R/pmcmc.R:645]: Did you want to remove this line?

File: R/parameters.R [hunk 3]
```diff
@@ -376,6 +376,17 @@ sircovid_date <- function(date) {
   days_into_2020
 }
 
+# Converts dates from from sircovid_Date as used in the MCMC to a Date
+# Automatically converts type
+sircovid_date_to_date <- function(sircovid_date)
```
Comment (johnlees) [inline R/parameters.R:381]: This should have a more informative name, perhaps `sircovid_date_as_Date()` to make the class of the return type clearer

File: NEWS.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+# sircovid 0.2.35
+
+- fix bug in pMCMC
```
Comment (johnlees) [inline NEWS.md:3]: 'Remove date offsets from pMCMC'

File: R/parameters.R (hunk 3 above)
Comment (johnlees) [reply R/parameters.R:381]: Also there are a two places in the plots where this same transform is used, I'd suggest switching these out to this function too:
- https://github.com/mrc-ide/sircovid/blob/master/R/sampler.R#L210
- https://github.com/mrc-ide/sircovid/blob/master/R/pmcmc.R#L644

File: R/pmcmc.R (hunk 2 above)
Comment (lwhittles) [reply R/pmcmc.R:645]: Yes, when we used the offsets the CrI on start_date would appear the wrong way round when we converted back to dates so this was just housekeeping to deal with that. Now that we use sircovid_dates the 2.5% and 97.5% CrIs no longer need to be switched

**Pull request merged**